rand_core/src/impls.rs

```diff
@@ -219,20 +219,38 @@ impl<R: BlockRngCore> BlockRng<R> {
     }
 
     /// Return a reference the wrapped `BlockRngCore`.
+    #[inline(always)]
```

Comment:
Just curious: did inlining turn out to be necessary, or did you just inline all methods?

Reply:
I haven't benchmarked at all, still meaning to do that.